README.md

```diff
@@ -82,7 +84,11 @@ You can connect to the admin interface at http://127.0.0.1:8000/admin with your
 
 # Support
 
-Make an issue in [issues](github.com/SustainableUrbanDesign/app/issues), check if you're having any issues like anyone else. If not then make an issue describing what happened, and what the error says.
+* See [How To Make an Issue](https://github.com/SustainableUrbanDesign/app/issues/35)
```

Comment:
Let's move the instructions into a proper contributing guide:
http://mozillascience.github.io/working-open-workshop/contributing/